db/migrate/20160617123557_add_meta_fields_to_theming.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetaFieldsToTheming < ActiveRecord::Migration
+  def change
+    add_column :pageflow_themings, :default_author, :string
+    add_column :pageflow_themings, :default_publisher, :string
+    add_column :pageflow_themings, :default_keywords, :string
```

Comment:
I'm wondering if we should set explicit defaults from the configuration for existing themings. Then we could remove the fallback logic completely. Otherwise we would have to display placeholders in the theming form.

On the other hand when we remove fallback logic we have to also migrate existing revisions